src/main/java/controller/TwitterApp.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import twitter4j.*;
+import twitter4j.conf.ConfigurationBuilder;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class TwitterApp {
+
+
+    public Twitter getTwitterInstance() {
+        ConfigurationBuilder cb = new ConfigurationBuilder();
+        cb.setDebugEnabled(true)
+                .setOAuthConsumerKey("**")
+                .setOAuthConsumerSecret("**")
+                .setOAuthAccessToken("**")
+                .setOAuthAccessTokenSecret("***");
+        TwitterFactory tf = new TwitterFactory();
+        Twitter twitter = tf.getInstance();
+        return twitter;
+    }
+
+
+    void getTimeline(Twitter twitter) throws TwitterException {
+
+        List<Status> statuses = twitter.getHomeTimeline();
+        for (Status st : statuses) {
+            System.out.println(st.getUser().getName() + "------" + st.getText()+"\n");
+        }
+    }
+
+    void postTweet(Twitter twitter, String msg) throws TwitterException {
+
+        Status status = twitter.updateStatus(msg);
+        System.out.println("POst tweeted");
+    }
+
+
+    public static void main(String[] args) throws TwitterException {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the message to be posted");
+        String st = sc.nextLine();
```

Comment:
Always give meaningful names for all variables. For example, here it could be `String postMessage = sc.nextLine();` 
Similarly assign other variable names like sc, td with meaningful ones. Most of the time intellij will suggest variable names, you can use those as long as it makes sense in your context.

Reply:
Okay,I will change.